Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -202,8 +214,11 @@ namespace AZ
             void OnAddedToScene(Scene* scene);
             void OnRemovedFromScene(Scene* scene);
 
+            // Called before this pipeline is about to be rendered and before SRGs are bound.
```

Comment:
question: Noob question but is SRGs defined anywhere in this file? I'm not sure what it is sorry 🙈 Might be worth having the full name once somewhere?